components/logins/src/error.rs

```diff
@@ -76,4 +76,6 @@ pub enum InvalidLogin {
     BothTargets,
     #[fail(display = "Neither `formSubmitUrl` and `httpRealm` are present")]
     NoTarget,
+    #[fail(display = "Some unspecified problem; no error details sorry :-(")]
+    Unspecified,
```

Comment:
Guessing we'll be able to add more errors here as we iron out more of the details about what failures we need, correct?

Reply:
Right. @lougeniaC64 is adding a better error type over in #2262 and we can expand it as necessary from there.